include/smack/SmackRep.h

```diff
@@ -94,6 +109,23 @@ class SmackRep {
   const Expr* b2i(const llvm::Value* v);
 
 public:
+  //
+  // Setter for BIT_VECTOR flag
```

Comment:
No need for self-explanatory comments such as "setter for...." or "getter for...."